book/website/community-handbook/bookdash/bookdash-application.md

```diff
@@ -0,0 +1,95 @@
+(ch-bookdash-application)=
+# Application and Review Process
+
+The application, review, and selection process have been developed to provide a fair opportunity for all the interested members to show their interest in attending our book dash events.
+
+In this subchapter, we have provided details on the book dash's application and reviewing process.
+
+## Call for application
+
+The book dash event is organised for up to 15 participants.
+The reason behind keeping it small is to allow the core team members to offer specific mentorship to the invited participants and make this an interactive, intimate, fun, and productive event.
+
+To fairly make this selection, a call for application is opened at least 3 months in advance so that interested members can share their skills, interest, and expected outcomes through a short application.
```

Comment:
```suggestion
To fairly make this selection, a call for applications is opened at least 3 months in advance so that interested members can share their skills, interest, and expected outcomes through a short application.
```